packages/manager/modules/pci/src/projects/project/storages/databases/components/order-review/translations/Messages_fr_FR.json

```diff
@@ -2,7 +2,7 @@
   "pci_database_order_review_title": "Votre commande",
   "pci_database_order_review_selected_database": "Base de données sélectionnée",
   "pci_database_order_review_selected_plan": "Offre sélectionnée",
-  "pci_database_order_review_selected_flavor": "Caractéristiques sélectionnées",
+  "pci_database_order_review_selected_flavor": "Dimensionnement selectionné",
```

Comment:
Seems weired for me, may be better like this :
`Dimension selectionnée` or `Dimensionnement selectionnée`

Reply:
`Dimensionnement selectionné` is correct.

Here we are not talking about the `size`. Meaning here should be the `resizing`.